src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Figure extends Drawable {
```

Comment:
delete this interface 
getColor can be it AbstractFigure (getter is not a behavior)